src/openrct2/interface/Console.cpp

```diff
@@ -254,14 +273,14 @@ void console_draw(rct_drawpixelinfo *dpi)
     gfx_draw_string(dpi, lineBuffer, TEXT_COLOUR_255, x, y);
 
     // Draw caret
-    if (_consoleCaretTicks < 15) {
+    if (_consoleCaretTicks < CONSOLE_CARET_FLASH_THRESHOLD) {
         memcpy(lineBuffer, _consoleCurrentLine, _consoleTextInputSession->SelectionStart);
         lineBuffer[_consoleTextInputSession->SelectionStart] = 0;
         sint32 caretX = x + gfx_get_string_width(lineBuffer);
         sint32 caretY = y + lineHeight;
 
-        uint8 caretColour = ColourMapA[BASE_COLOUR(backgroundColour)].dark;
-        gfx_fill_rect(dpi, caretX, caretY, caretX + 6, caretY + 1, caretColour);
+        uint8 caretColour = ColourMapA[BASE_COLOUR(backgroundColour)].lightest;
```

Comment:
We use a lighter colour and thinner style for the caret in `TextInput.cpp`. This may be a nice opportunity to unify them, either way.

Reply:
Good point.